index.js

```diff
@@ -7,16 +8,16 @@ const doctrine = require('doctrine')
 const JSDOC_MERMAID_TAG = /@mermaid\b/
 
 /**
- * @constant {string} MERMAID_VERSION
- * @description Semver Mermaid version
+ * @constant {string} MERMAID_HTML_SCRIPT
+ * @description Html tag that includes mermaid library
  */
-const MERMAID_VERSION = '8.4.8'
+const MERMAID_HTML_SCRIPT = '<script src="https://unpkg.com/mermaid${MERMAID_VERSION}/dist/mermaid.min.js"></script>'
 
 /**
- * @constant {string} MERMAID_HTML_SCRIPT
- * @description Html tag that include mermaid library
+ * @constant {string} MERMAID_INIT_SCRIPT
+ * @description Html script that include initializes Mermaid with config options
  */
-const MERMAID_HTML_SCRIPT = `<script src="https://unpkg.com/mermaid@${MERMAID_VERSION}/dist/mermaid.min.js"></script>`
+const MERMAID_INIT_SCRIPT = '<script>mermaid.initialize(${MERMAID_CONFIG});</script>\n'
```

Comment:
This is the suggestion i'm making in my comment from #5 

**BEFORE**:
```
const MERMAID_INIT_SCRIPT = '<script>mermaid.initialize(${MERMAID_CONFIG});</script>\n'
```

**AFTER**:
```
const MERMAID_INIT_SCRIPT = '<script type=module>

  mermaid.initialize(${MERMAID_CONFIG});

  // Get major version as a number
  const MAJOR_VERSION= parseInt(MERMAID_VERSION.split('.')[0])

  // Start Mermaid after page has fully loaded
  if(MERMAID_VERSION >= 10){
    await mermaid.run();
  }
  else{
    mermaid.init();
  }
</script>\n'

```

Reply:
@cmidgley i'd think we can prolly include this change before merging. What are your thoughts? 

I think we can slightly tweak it.  

```
const MERMAID_INIT_SCRIPT = '<script type=module>
  mermaid.initialize(${MERMAID_CONFIG});

  if (typeof mermaid.run === 'function') {
    await mermaid.run();
  } else {
    mermaid.init();
  }
</script>\n'